### PR TITLE
Added inform and nestedtext packages.

### PR DESCRIPTION
**Pull request by KenKundert**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by KenKundert** [comment]
@conda-forge/help-python, hopefully ready to merge!

**Comment by KenKundert** [comment]
@conda-forge/help-python, ready to merge

File: recipes/inform/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/inform-{{ version }}.tar.gz
+  sha256: b80f9e8873731f0bcd9734ed9124d79dda8b9748c7e3e17c6f70b02cd4390bd2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/inform/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipes/nestedtext/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nestedtext-{{ version }}.tar.gz
+  sha256: 8837596bdb5936b05ead4517713dc67f8118760ef59e58777475d4a8dd20cbac
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/nestedtext/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

**Comment by KenKundert** [comment]
The nestedtext package depends on the inform package, but nestedtext was added first.  As a result, the tests failed because inform was not available.  Now inform is available, but nestedtext is not.  I feel if I could restart the tests the issue would resolved.  But I can not find any way to restart the tests.  The Azure documentation shows a "rerun the tests" button, but I can find no such button.  I have read the documentation and I can find nothing on how to resolve this. Can you help me?

**Comment by ocefpaf** [comment]
All you need is to restart the CIs. You should have permission to do that as a maintainer but I'll give it a go now.

**Pull request merged**